### DISCUSSION TITLE
1.9 won't install on W2K Sp4

### DISCUSSION THREAD

**Discussion by thellmer-source** [LegacyUpdate/LegacyUpdate#264] [Questions & Help]
I have a newly restored IBM PC-350 p-166 and a fresh install of W2kProf SP4.  I can install LU up through 1.7 without error but nothing really ever checks for updates and it never places any icons anywhere, but at least installing those older versions  never error out.  However trying to go through the LU site and install the new update, or even downloading the exe from scratch gives me the following same install error.  Trying to install 1.8 or 1.9 directly or indirectly, gives me an error "LegacyUpdate-1.exe has generated errors and will be closed..." error.  It says an error log is being generated but I can't seem to find one anywhere.  I was able to use the WindowsUpdateTestored v5 site to get what I believe to be all updates, so probably not a big issue...but I like the more seamless LU V6 option that I get in XP and Vista on my other old dual boot PC and which seems to be an option in W2K through LU if it would just install.  I do have MS Office 2003 SP3 that W2Restored finds updates for but won't install, whereas on my XP and Vista drives LU found and installed those correctly.  Any idea what to start to look at?  I installed W2KProf on one of the old original 1.2 GB drives which was a little small but worked, but I threw in another new 4GB drive and another fresh W2KProf install on it which now boots as my default, but I get the same error on both W2K installs.  WindowsUpdateRestored V5 did find and install about 130 updates, (other than the MS Office ones) so I feel like I am probably updated now, but just curious why LU isn't working.  LU works fine on my other PC with both an XP and Vista install.  I have good internet on the IBM, and WUPRestored seems to work fine.  It has 3 ATA/IDE drives with an old NT4 being the master menu boot, but then passes the OS through boot.ini to the other W2K installs to load based on selection or default.  If I knew where to find the supposed log file exists I could see what it contains if that would help.  Both installs are Sp4 with IE6 SP1 with VC++ and .net 2 additions.   Ideas?

**Comment by kirb** [comment]
Sorry for the issue! This is a bug with 1.8 and 1.9 indeed. I changed some things about how the setup is compiled, and that unexpectedly made it depend on instructions only available in Pentium Pro/II/III and later. 1.10 will fix it, but until then you should be fine sticking to 1.7.1.

**Reply by thellmer-source** [reply]
Ah thanks!  That explains it :-)
